packages/admin-panel/src/pages/resources/ProjectsPage.js

```diff
@@ -51,10 +51,11 @@ const FIELDS = [
     type: 'tooltip',
   },
   {
-    Header: 'Tile Sets',
-    source: 'tile_sets',
-    secondaryLabel: 'Comma separated list (eg. osm,satellite,terrain).',
-    type: 'tooltip',
+    Header: 'Config',
+    source: 'config',
+    type: 'jsonTooltip',
+    editConfig: { type: 'jsonEditor' },
+    secondaryLabel: 'eg. {tileSets: osm,satellite,terrain, stickyRegionLabels: true}',
```

Comment:
```suggestion
    secondaryLabel: 'eg. {"tileSets": ["osm","satellite","terrain"], "stickyRegionLabels": true}',
```

Reply:
Oh, looking at the tests perhaps it's not a fully formatted array - even more important to specify I reckon!